### ISSUE TITLE
Stuck in air when mounted and entering new zones (flying)

### ISSUE THREAD

**Issue by Rektnready** [dalaranwow/dalaran-wow#5882]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Can recreate the bug now happens almost everytime I fly from dalaran teleporter to storm peaks just as I enter the zone my character gets stuck, unable to move except to turn in place, movement animation of mount continues, if I dismis the mount I am literally stuck in running animation in mid air. The only way to get out is to alt-f4 but when i relog i fall out of the sky and die meaning repair costs. This is happening several times a day everyday. If I am in que for rhc I cannot tele to instance and have to leave to relog and die. Also happens (though less often) when hs/astral recall to dalaran, get stuck in place and have to relog.

![WoWScrnShot_042120_140223](https://user-images.githubusercontent.com/64081095/79869106-d2247280-83d8-11ea-9e3b-2530e105565a.jpg)
![WoWScrnShot_042120_140215](https://user-images.githubusercontent.com/64081095/79869136-dd779e00-83d8-11ea-830f-41c565a5a3c4.jpg)

**Expected behaviour**: simple, normal transition between zones please.
**Steps to reproduce the problem**:

1. For me the highest chance is to tele below dalaran
2. fly to storm peaks on flying mount
3. get stuck

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills** using black proto drake in screens but happens on any mount.

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
